src/RpcNep17Tracker/RpcNep17Tracker/config.json

```diff
@@ -1,8 +1,9 @@
 {
   "PluginConfiguration": {
     "DBPath": "Nep17BalanceData",
-    "TrackHistory" : true,
+    "TrackHistory": true,
     "RecordNullAddressHistory": false,
-    "MaxResults" : 1000
+    "MaxResults": 1000,
+    "Active": 5195086
```

Comment:
This value should be 0 by default for all plugins, isn't it?

Reply:
This is the MainNet magic number.

Reply:
Then we should rename it to something more descriptive, `Network` ?

Reply:
Renamed.